tracer/src/Datadog.Trace.Trimming/build/Datadog.Trace.Trimming.xml

```diff
@@ -928,7 +928,6 @@
       <type fullname="System.Text.RegularExpressions.Match" />
       <type fullname="System.Text.RegularExpressions.MatchCollection" />
       <type fullname="System.Text.RegularExpressions.Regex" />
-      <type fullname="System.Text.RegularExpressions.RegexMatchTimeoutException" />
```

Comment:
Unsure if this was removed the last time, but just wanted to check to see if this is right that it is removed?
I'd imagine so as I think this file is just auto generated, but just want to double check

Reply:
Hmm, good catch. There was `catch` specifically for `RegexMatchTimeoutException` [here](https://github.com/DataDog/dd-trace-dotnet/blob/853f2a8625b7f471d4d797bf0a425fb7c1c78e27/tracer/src/Datadog.Trace/Sampling/CustomSamplingRule.cs#L123) which I removed. I think that's what caused this line to be removed.

It seems weird because there's another unrelated `catch` [here](https://github.com/DataDog/dd-trace-dotnet/blob/853f2a8625b7f471d4d797bf0a425fb7c1c78e27/tracer/src/Datadog.Trace/Util/Http/QueryStringObfuscation/Obfuscator.cs#L58).

What's interesting is that both places not choose between `System.Text.RegularExpressions` or `Datadog.Trace.Vendors.IndieSystem.Text.RegularExpressions` depending on the target framework, but the XML file doesn't specify TFMs.

Reply:
Oh, trimming is only supported in newer .NET versions, and we use `Datadog.Trace.Vendors.IndieSystem.Text.RegularExpressions` there, so I think it may be correct to remove `System.Text.RegularExpressions.RegexMatchTimeoutException`.